core/src/main/java/com/microsoft/applicationinsights/internal/persistence/PersistenceHelper.java

```diff
@@ -0,0 +1,89 @@
+package com.microsoft.applicationinsights.internal.persistence;
+
+import com.microsoft.applicationinsights.internal.util.LocalFileSystemUtils;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.FilenameUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Collection;
+import java.util.UUID;
+
+final class PersistenceHelper {
+
+    private static final Logger logger = LoggerFactory.getLogger(PersistenceHelper.class);
+
+    private static final long MAX_FILE_SIZE_IN_BYTES = 52428800; // 50MB
```

Comment:
is this what max size used to be? for some reason i recall 10mb

Reply:
this is what we have in main branch: 
 public final static int DEFAULT_CAPACITY_MEGABYTES = 10;
 private final static int MAX_CAPACITY_MEGABYTES = 1000; 
 private final static int MIN_CAPACITY_MEGABYTES = 1;

dotnet sdk is 50MB and same for otel-dotnet.

Reply:
ok, so our default was 10mb, but makes sense to me to align with the dotnet default, let's check default with Python and Node also